README.md

```diff
@@ -1,5 +1,127 @@
 # DogStatsD.Interceptors
 
+A castle proxy interceptor that branches out in to multiple interceptors to push metrics back to DataDog via DogStatD.
+
+[![install from nuget](http://img.shields.io/nuget/v/DogStatsD.Interceptors.svg?style=flat-square)](https://www.nuget.org/packages/DogStatsD.Interceptors)
+[![downloads](http://img.shields.io/nuget/dt/DogStatsD.Interceptors.svg?style=flat-square)](https://www.nuget.org/packages/DogStatsD.Interceptors)
+[![Build status](https://ci.appveyor.com/api/projects/status/7c3jrhlnp0bvph95/branch/master?svg=true)](https://ci.appveyor.com/project/Liberis/dogstatsd-interceptors/branch/master)
+
+## Getting Started
+
+DogStatsD.Interceptors is split in to 2 packages, `DogStatsD.Interceptors` and `DogStatsD.Interceptors.Annotations`. `DogStatsD.Interceptors` is the core implemtation while `DogStatsD.Interceptors.Annotations` containing only Attributes which act as method markers.
+
+DogStatsD.Interceptors can be installed via the package manager console by executing the following commandlet:
+
+```powershell
+PM> Install-Package DogStatsD.Interceptors
+```
+
+Once we have the package installed, we need to setup the DogStatsD client for c#, this is achieved at start of your application by calling `Configure` on `DogStatsd` object:
```

Comment:
Not sure 1st person is appropriate for a readme. Consider instead:

"Once the package is installed, you need to setup the DogStatD client for C#. Call the `Configure` method on the `DogStatsd` object as follows"